### ISSUE TITLE
Replace content under "Important Articles"

### ISSUE THREAD

**Issue by UmerManager** [TaoFruit/reveltek#34]
Website Link: https://reveltek.com/index.html#

**Instructions to follow:**
* Replace Title's content and three sections under it with suggested content:
  * IGNITE CURIOUSTY
  * SMILE METHOD
  * HIVE MENTALITY 
* You have to merge these three titles with headlines and describe content accordingly.
* You can visit our Reveltek Handbook to get better understanding for these headlines, link is shared below here.

* https://focushive.sharepoint.com/sites/Everyone/Shared%20Documents/General/Reveltek-Handbook%20(2).pdf 

* Please find the attached image for reference:

![image](https://user-images.githubusercontent.com/111144473/207333624-8c87c2b2-060b-4658-8cbf-9fa589d41c92.png)

**Comment by wise-guru** [comment]
I would like to work on this. Is there a way to view the handbook without having an account?

**Comment by UmerManager** [comment]
please find attached pdf file of Reveltek Handbook.

Regards,

Umer Farooq


________________________________

**Comment by Aiko-Kawamura** [comment]
Hello @UmerManager. Do we need to create a .md file of research on this repo?

**Comment by UmerManager** [comment]
Yes, create a .md file if it is not existing already and save it in the repo in "Notes" folder
________________________________

**Comment by Aiko-Kawamura** [comment]
https://github.com/Aiko-Kawamura/Research-for-Important-Article-section.git

**Comment by SanaGhouri09** [comment]
[Article Ideas for Reveltek Website.md](https://github.com/TaoFruit/reveltek/files/10374413/Article.Ideas.for.Reveltek.Website.md)
